src/platforms/python/guides/fastapi/index.mdx

```diff
@@ -40,6 +38,8 @@ sentry_sdk.init(
 app = FastAPI()
 ```
 
+The Sentry SDK automatically enables the FastAPI integration if you have the `fastapi` Python package installed in your project. There are no configuration options you need to add when initializing the Sentry SDK as everything works out of the box.
```

Comment:
I moved it down because it is only a implementation detail. The important thing is the code snippets, if you need to set an configuration option or not is kind of not relevant because you have the code that shows how to do it. 

And I did not want to have 4 lines of intro text (that most people will just skip over). This way it is also showing more about the install/configure on mobiles phones on loading of the page.

Reply:
Is it ok for you to keep this below the code snippet, or do you have a strong wish to have it at the very top below the first line?

Reply:
Great point about the mobile view! But I'm wondering, if a person already has the Python package installed, it sounds like they don't need to do any of the steps we talk about below (install or configure), right? So to me, it makes sense to keep the information about the Python package at the top so people who have Python installed can stop reading. Always happy to jump on a chat and talk more :) I'm brand new so still have a lot to learn!

Reply:
Oooh! Now I get your point of view! It is amazing that someone that is not used to our documentation reads it. The explanation with the auto detection has actually nothing do to with the install or configure step. (When auto enabling stuff you do not have to give the "integrations" parameter to the `init()`. 

So I think this paragraph will confuse a lot of people and we should just remove it. (It is  basically a relic of the time when we did not have auto enabling, and everyone had to set the integrations they want to enable.)